CleanWeather/Scenes/CitiesList/CitiesListController.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  CitiesListController.swift
```

Comment:
zazwycaj jak masz LIST to samo List wskazuje na liczbę mnogą, dlatego lepiej będzie mieć cały moduł nazwany CityList

bo dzieki temu poteem mozesz miec np moduly:

CityDetails
CityAdd
CityEdit itp

i masz wspólny mianownik 'City' oznaczajacy taką grupę

Reply:
Nazwy modułu zmienione na City...